wordpress-theme/skyyrose-flagship/assets/css/product-grid.css

```
@@ -40,16 +40,21 @@
 	grid-row: span 2;
 }
-.product-grid__items .product-card:nth-child(4n+1) .holo-card__media {
+/* Fixed: was .holo-card__media (dead selector); correct class is .holo__gallery */
+.product-grid__items .product-card:nth-child(4n+1) .holo__gallery {
 	aspect-ratio: auto;
 	height: 100%;
 }
 
 @media (max-width: 1024px) {
-	.product-grid__items { grid-template-columns: repeat(2, 1fr); }
+	.product-grid__items { grid-template-columns: repeat(2, 1fr); gap: 24px; }
```

Stylelint (17.14.0)

[error] Too many declarations, maximum 1 (declaration-block-single-line-max-declarations)

(declaration-block-single-line-max-declarations)